pypmc/mix_adapt/hierarchical.py

```diff
@@ -5,6 +5,9 @@
 import numpy as np
 import scipy.linalg as linalg
 
+import logging
+logger = logging.getLogger(__name__)
```

Comment:
This creates sub loggers, with a name of ``pypmc.mix_adapt.hierarchical``, correct?
Is there an easy way to get to all loggers that belong to the ``pypmc`` root logger?

Reply:
Correct.
Checking the logging documentation, I don't think it's designed such that you can get a list of all child loggers. Instead, you are supposed to configure the parent logger (`pypmc` or `pypmc.mix_adapt`). This has the advantage that child loggers, which are instantiated later also adhere to the parent.

Reply:
Weird, but apparently the recommended approach.